bundles/org.openhab.binding.livisismarthome/src/main/java/org/openhab/binding/livisismarthome/internal/handler/LivisiBridgeHandler.java

```diff
@@ -821,6 +831,19 @@ public void commandSetRollerShutterStop(final String deviceId, final ShutterActi
                 (capabilityId) -> client.setRollerShutterAction(capabilityId, action));
     }
 
+    /**
+     * Restarts the SHC (bridge) device
+     */
+    public void commandRestart() {
+        try {
+            client.setRestartAction(bridgeId);
+
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE, "Restarting ...");
```

Comment:
Status detail **BRIDGE_OFFLINE** is used to indicate that the parent bridge if offline, not the bridge itself. See [here](https://www.openhab.org/docs/concepts/things.html#status-details).
```suggestion
            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE, "Restarting...");
```

Reply:
done